index.html

```diff
@@ -108,7 +108,13 @@ <h4>Keep in touch with us!</h4>
 					target="_blank"
 					rel="noopener"
 					>GitHub</a
-				>
+				><a
+					class="homepage-link"
+					href="https://about.ifelse.network/contact.html"
```

Comment:
the HREF should use a relative path to enable local development and subdomain moving E.g. `href="/contact.html"`.
We should also avoid having filename extensions in URLs because it looks unprofessional, so if we renamed `contact.html` to `contact/index.html` the HREF would look like this: `href="/contact"`.